FORMULA

```diff
@@ -1,9 +1,9 @@
 name: eclipse
 os: Debian, Ubuntu, Raspbian, RedHat, Fedora, CentOS, Amazon, Suse, openSUSE, Gentoo, Funtoo, Arch, Manjaro, Alpine, FreeBSD, OpenBSD, Solaris, SmartOS, Windows, MacOS
 os_family: Debian, RedHat, Suse, Gentoo, Arch, Alpine, FreeBSD, OpenBSD, Solaris, Windows, MacOS
-version: 1.0.0
+version: 0.2.2
```

Comment:
Once `semantic-release` is enabled, this line should never be changed manually.  I'll try to resolve that in the commit to be added.